<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInvoiceDialog</class>
 <widget class="QDialog" name="EditInvoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sửa Hóa Đơn</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Thông tin hóa đơn -->
   <item>
    <widget class="QGroupBox" name="invoiceGroup">
     <property name="title">
      <string>Thông tin hóa đơn</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mã hóa đơn:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="invoiceId">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mã khách hàng:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="customerId"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ngày lập:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="invoiceDate">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <!-- Danh mục và danh sách sản phẩm -->
   <item>
    <widget class="QGroupBox" name="productGroup">
     <property name="title">
      <string>Danh sách sản phẩm</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Danh mục:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categoryCombo">
          <item>
           <property name="text">
            <string>Thú cưng</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sản phẩm</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Số lượng:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="quantitySpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="productTable">
        <property name="minimumHeight">
         <number>200</number>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addToInvoiceButton">
          <property name="text">
           <string>Thêm vào hóa đơn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Chi tiết hóa đơn -->
   <item>
    <widget class="QGroupBox" name="detailGroup">
     <property name="title">
      <string>Chi tiết hóa đơn</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="detailTable">
        <property name="minimumHeight">
         <number>200</number>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="editDetailButton">
          <property name="text">
           <string>Sửa chi tiết</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteDetailButton">
          <property name="text">
           <string>Xóa chi tiết</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalLabel">
          <property name="text">
           <string>Tổng tiền:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="totalAmount">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Buttons -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Hủy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>